versioning.gradle

```diff
@@ -35,11 +35,19 @@ ext {
         }
     }
 
+    getStoredTag = {
+        if(project.hasProperty("latestTag")) {
```

Comment:
if the property is present we use that value, in local builds whilst developing we would be getting the tag from github directly